### PR TITLE
Chore(demo): Hide elements in Supernova component preview

### PR DESCRIPTION
**Pull request by curdaj**

### CODE REVIEW

File: apps/demo/netlify.toml [hunk 1]
```diff
@@ -34,3 +34,10 @@ ignore = """\
     ./packages/web/ \
     ./.gihub/workflows/ \
     """
+
+[[headers]]
+# you can configure the correct paths here
+for = "/*"
+[headers.values]
+# see a list of allowed values here: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin
+access-control-allow-origin = "*"
```
Comment (literat) [inline apps/demo/netlify.toml:43]: **suggestion (if-minor):** This is ok for testing. After we check that this is working, can we please add the actual domain?

**Review by literat — APPROVED** [review]

File: apps/demo/netlify.toml (hunk 1 above)
Comment (curdaj) [reply apps/demo/netlify.toml]: outdated

File: apps/demo/partials/layout/base.hbs [hunk 2]
```diff
@@ -46,6 +46,19 @@
         });
         targetElement.classList.add(`theme-${theme}`);
       };
+
+      // Hide elements with hide-from-visual-tests class in Supernova component preview via postMessage
+      window.addEventListener('message', (event) => {
+          // Check if the postMessage is from the Supernova.io domain
+          if (event.origin === 'https://app.supernova.io') {
```
Comment (literat) [inline apps/demo/partials/layout/base.hbs:53]: **suggestion (if-minor):** The condition can be combined, IMHO.

File: apps/demo/partials/layout/base.hbs [hunk 3]
```diff
@@ -46,6 +46,19 @@
...
+              if (event.data.action === 'hide-from-visual-tests') {
+                  const elements = document.querySelectorAll('.hide-from-visual-tests');
+                  elements.forEach(el => {
```
Comment (literat) [inline apps/demo/partials/layout/base.hbs:56]: **nitpick (if-minor):** 
```suggestion
                  elements.forEach(element => {
```

File: apps/demo/partials/layout/base.hbs [hunk 4]
```diff
@@ -46,6 +46,19 @@
         });
         targetElement.classList.add(`theme-${theme}`);
       };
+
+      // Hide elements with hide-from-visual-tests class in Supernova component preview via postMessage
```
Comment (literat) [inline apps/demo/partials/layout/base.hbs:50]: **nitpick (if-minor):** Can you please add a link to supernova.io settings so we can find a place where it is used? With a proper comment, of course.

**Review by literat — APPROVED** [review]

**Pull request merged**